control/control.go

```diff
@@ -127,6 +127,8 @@ type managesPlugins interface {
 	SetMetricCatalog(catalogsMetrics)
 	GenerateArgs(logLevel int) plugin.Arg
 	SetPluginConfig(*pluginConfig)
+	SetPluginTags(map[string]map[string]string)
+	AddStandardAndWorkflowTags(core.Metric, map[string]map[string]string) core.Metric
```

Comment:
Short answer: test won't pass.

Global tags are stored in the pluginManager; which is a `managesPlugin` interface too.